python/src/iceberg/expressions/base.py

```diff
@@ -88,7 +89,7 @@ def negate(self) -> "Operation":
 }
 
 
-class Literal(Generic[T], ABC):
+class Literal(Generic[T], metaclass=ABCMeta):
```

Comment:
Why do we want the reader to know that this is changing a metaclass and not just inheriting from ABC? It seems like bringing a metaclass into the code is more clear about what's happening, but introduces a distinction that people (like me) may not understand or need.